mbofui/app/js/controllers.js

```diff
@@ -204,10 +204,10 @@ ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter
     // use a promise factory to do request
     Bof.GetBofs(bofsUrl).then(function(eventsList) {
       Bof.GetIntentions($rootScope.urls.intentions + '?username=self').then(function(intentionsList) {
-        $scope.intentions=intentionsList;
+        $scope.intentions=intentionsList.data.results;
```

Comment:
intentionList has a lot of info added by Angular. Here we strip everything but the actual data we are interested in

```
{
	"data": {
		"count": 4,
		"next": null,
		"previous": null,
		"results": [{
			"url": "http://localhost:18000/api/intentions/1/",
			"intention": "maybe",
			"respondent": "http://localhost:18000/api/users/bjensen/",
			"event": "http://localhost:18000/api/events/106/"
		}]
	},
	"status": 200,
	"config": {
		"method": "GET",
		"transformRequest": [null],
		"transformResponse": [null],
		"url": "/api/intentions/?username=self",
		"headers": {
			"Accept": "application/json, text/plain, */*"
		}
	},
	"statusText": "OK"
}
```